tests/fixtures/expected_tile_sql_complex_feature_push_down_eligible.sql

```diff
@@ -0,0 +1,215 @@
+          "T0"."_fb_internal_item_sum_item_amount_event_id_col_None_join_1" AS "_fb_internal_item_sum_item_amount_event_id_col_None_join_1"
+        FROM (
+          SELECT
+            L."col_int" AS "col_int",
+            L."col_float" AS "col_float",
+            L."col_char" AS "col_char",
+            L."col_text" AS "col_text",
+            L."col_binary" AS "col_binary",
+            L."col_boolean" AS "col_boolean",
+            L."event_timestamp" AS "event_timestamp",
+            L."cust_id" AS "cust_id",
+            R."col_float" AS "col_float_scd",
+            R."col_binary" AS "col_binary_scd",
+            R."col_boolean" AS "col_boolean_scd",
+            R."created_at" AS "created_at_scd",
+            R."cust_id" AS "cust_id_scd"
+          FROM (
+            SELECT
+              "__FB_KEY_COL_0",
+              "__FB_LAST_TS",
+              "col_int",
+              "col_float",
+              "col_char",
+              "col_text",
+              "col_binary",
+              "col_boolean",
+              "event_timestamp",
+              "cust_id"
+            FROM (
+              SELECT
+                "__FB_KEY_COL_0",
+                LAG("__FB_EFFECTIVE_TS_COL") IGNORE NULLS OVER (PARTITION BY "__FB_KEY_COL_0" ORDER BY "__FB_TS_COL", "__FB_TS_TIE_BREAKER_COL") AS "__FB_LAST_TS",
+                "col_int",
+                "col_float",
+                "col_char",
+                "col_text",
+                "col_binary",
+                "col_boolean",
+                "event_timestamp",
+                "cust_id",
+                "__FB_EFFECTIVE_TS_COL"
+              FROM (
+                SELECT
+                  CAST(CONVERT_TIMEZONE('UTC', "event_timestamp") AS TIMESTAMP) AS "__FB_TS_COL",
+                  "cust_id" AS "__FB_KEY_COL_0",
+                  NULL AS "__FB_EFFECTIVE_TS_COL",
+                  2 AS "__FB_TS_TIE_BREAKER_COL",
+                  "col_int" AS "col_int",
+                  "col_float" AS "col_float",
+                  "col_char" AS "col_char",
+                  "col_text" AS "col_text",
+                  "col_binary" AS "col_binary",
+                  "col_boolean" AS "col_boolean",
+                  "event_timestamp" AS "event_timestamp",
+                  "cust_id" AS "cust_id"
+                FROM (
+                  SELECT
+                    "col_int" AS "col_int",
+                    "col_float" AS "col_float",
+                    "col_char" AS "col_char",
+                    "col_text" AS "col_text",
+                    "col_binary" AS "col_binary",
+                    "col_boolean" AS "col_boolean",
+                    "event_timestamp" AS "event_timestamp",
+                    "cust_id" AS "cust_id"
+                  FROM "sf_database"."sf_schema"."sf_table"
+                  WHERE
+                    "event_timestamp" >= CAST(__FB_START_DATE AS TIMESTAMPNTZ)
+                    AND "event_timestamp" < CAST(__FB_END_DATE AS TIMESTAMPNTZ)
```

Comment:
This filter is now applied at a lower level here when selecting from event table.